Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -125,19 +135,23 @@ open class NavigationMapView: UIView {
      
      The `UserCourseView`'s `UserCourseView.update(location:pitch:direction:animated:)` method is frequently called to ensure that its visual appearance matches the map’s camera.
      */
-    public var userCourseView: UserCourseView = UserPuckCourseView(frame: CGRect(origin: .zero, size: 75.0)) {
```

Comment:
`NavigationMapView.userCourseView` is now used only internally to prevent issues when user can modify both `NavigationMapView.userCourseView` and `NavigationMapView.userLocationStyle`.

Reply:
As long as this is a stored property, it has the potential to get out of sync with the associated value of `UserLocationStyle.courseView(_:)` when that’s the value of the `userLocationStyle` property. Can we make this a computed property or eliminate it somehow?

Whether we remove `userCourseView` or make it private, it’s effectively gone to the developer, so we should note the removal in the changelog.